sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -262,16 +263,21 @@ def get_named_nested_pvalues(pvalueish):
     else:
       tagged_values = enumerate(pvalueish)
   elif isinstance(pvalueish, list):
+    if as_inputs:
+      yield None, pvalueish
```

Comment:
What's the rationale for this branch i.e. yielding the whole `pvalueish` versus enumerating it?

Reply:
This is for backwards compatibility with the "eager" mode (e.g. `[1, 2, 3] | beam.Map(lambda x: x*x)`)